test/javascripts/acceptance/param-input-test.js

```diff
@@ -304,7 +304,18 @@ acceptance("Data Explorer Plugin | Param Input", function (needs) {
     await fillIn(".query-params input", monthsAgoValue);
     await click("form.query-run button");
 
-    const searchParams = new URLSearchParams(currentURL());
```

Comment:
About this: the result of `Array.from(new URLSearchParams('/something?id=1&params={"a":1}').entries())` is actually `[ ['/something?id', '1'], ['params', '{"a":1}']]`

So the original logic is actually misleading. :face_holding_back_tears: